api/src/main/java/org/pmiops/workbench/reporting/insertion/UserColumnValueExtractor.java

```diff
@@ -72,7 +72,20 @@
   INSTITUTIONAL_ROLE_ENUM(
       "institutional_role_enum", u -> enumToString(u.getInstitutionalRoleEnum())),
   INSTITUTIONAL_ROLE_OTHER_TEXT(
-      "institutional_role_other_text", ReportingUser::getInstitutionalRoleOtherText);
+      "institutional_role_other_text", ReportingUser::getInstitutionalRoleOtherText),
+  DISABILITY("disability", u -> enumToString(u.getDisability())),
+  DEGREES("degrees", ReportingUser::getDegrees),
+  ETHNICITY("ethnicity", u -> enumToString(u.getEthnicity())),
+  GENDER_IDENTITY("gender_identity", ReportingUser::getGenderIdentity),
+  HIGHEST_EDUCATION("highest_education", u -> enumToString(u.getHighestEducation())),
+  IDENTIFIES_AS_LGBTQ(
+      "identifies_as_lgbtq",
+      u -> u.getIdentifiesAsLgbtq() == null ? null : u.getIdentifiesAsLgbtq().toString()),
```

Comment:
opt_nit: most of this code would probably use this style: `Optional.ofNullable(getIdentifies...).map(Lgbtq::toString).orElse(null)`, but I don't feel strongly

Reply:
Honestly, that is harder to read, I will just leave it since you don't have strong feeling... I like `use == ?a :b` style more.